.travis.yml

```diff
@@ -5,25 +5,21 @@ services:
 
 before_install:
 - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-- curl https://toolbelt.heroku.com/install.sh | sh  
-- echo "$HEROKU_API_KEY" | docker login -u "$HEROKU_USERNAME" --password-stdin registry.heroku.com
 
 script:
 - docker build --target test --tag todo-app:test .
 - docker run --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:test  todo_app/tests/
-- docker run --env GITHUB_REDIRECT_URI --env GITHUB_CLIENT_ID --env GITHUB_CLIENT_SECRET --env ROLEWRITER_USER --env MONGO_USERNAME --env MONGO_PASSWORD --env MONGO_DB --env MONGO_URL --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:test  todo_app/tests_e2e/
+- docker run --env GITHUB_REDIRECT_URI --env GITHUB_CLIENT_ID --env GITHUB_CLIENT_SECRET --env ROLEWRITER_USER --env MONGO_USERNAME --env MONGO_PASSWORD --env MONGO_DB --env MONGO_URL --env MONGO_OPTIONS --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:test  todo_app/tests_e2e/
 
 before_deploy:
 - docker build --target production --tag todo-app:prod .
 - docker tag todo-app:prod "$DOCKER_USERNAME"/todo-app:latest
-- docker tag todo-app:prod registry.heroku.com/$HEROKU_APP/web
 
 deploy:
 - provider: script 
   skip_cleanup: true 
   script:   
-      docker push "$DOCKER_USERNAME"/todo-app;      
-      docker push registry.heroku.com/$HEROKU_APP/web; 
-      heroku container:release web --app $HEROKU_APP;
-  on:
+      docker push "$DOCKER_USERNAME"/todo-app;
+      curl -dH -X POST "$AZURE_DEPLOY_WEBHOOK"
```

Comment:
You may find this doesn't actually work as expected.

Travis is a bit weird and processes the deploy script an extra time, so $AZURE_DEPLOY_WEBHOOK gets replaced with its value, and then the shell sees the dollar sign in the value and tries to interpret that as an environment variable.

If you just move these commands into a `deploy.sh` file, it will work.